### DISCUSSION TITLE
F3F8F05D-1A45-BDAB-12FD-03AF2D7260C1

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#105] [General]
# PSR B1828-11

PSR B1828-11 is a pulsar about 10,264 light-years away in the constellation Scutum. In 2006, it was theorized that a Pluto-sized object made out of quarks is orbiting only 65,000 km away from the neutron star. This hypothesis was listed in SIMBAD as controversial

https://celestia.mobi/addon?item=F3F8F05D-1A45-BDAB-12FD-03AF2D7260C1

**Comment by TheLostProbe** [comment]
"This hypothesis was listed in SIMBAD as controversial" hmm i wonder why 🤔
